modules/coin/handler.go

```diff
@@ -144,13 +147,47 @@ func (h Handler) sendTx(ctx sdk.Context, store state.SimpleDB,
 			ibcCtx := ctx.WithPermissions(ibc.AllowIBC(NameCoin))
 			_, err := cb.DeliverTx(ibcCtx, store, packet.Wrap())
 			if err != nil {
-				return err
+				return res, err
 			}
 		}
 	}
 
+	// now we build the tags
+	tags := make([]*abci.KVPair, 3)
+	tags[0] = tagInt("height", int64(ctx.BlockHeight()))
+
+	var names []string
+	for _, in := range send.Inputs {
+		addr := in.Address.Address.String()
+		names = append(names, addr)
+	}
+	tags[1] = tagStr("coin.sender", strings.Join(names, ","))
+
+	names = []string{}
+	for _, out := range send.Outputs {
+		addr := out.Address.Address.String()
+		names = append(names, addr)
+	}
+	tags[2] = tagStr("coin.receiver", strings.Join(names, ","))
+
 	// a-ok!
-	return nil
+	return sdk.DeliverResult{Tags: tags}, nil
+}
+
+func tagInt(key string, val int64) *abci.KVPair {
```

Comment:
oh.. these can go to ABCI

Reply:
Would be great! And rename them :)

Reply:
https://github.com/tendermint/abci/pull/144